### DISCUSSION TITLE
Welcome to splitpanes Discussions!

### DISCUSSION THREAD

**Discussion by antoniandre** [antoniandre/splitpanes#198] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by elarbor** [comment]
太棒的插件！
![动画](https://github.com/user-attachments/assets/a24d4cf0-2503-406d-a86a-2bc49ae87a1d)
